examples/gradle-kt-dsl/run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Retry the build for 30 minutes, every 10 seconds
+NUMBER_OF_RETRIES=180
+DELAY_BETWEEN_RETRIES=10
```

Comment:
what would cause the build to fail? and why 10s between build attempts?

Reply:
The build is going to be executed after artifacts are released to Maven. At this point, I don't know how long it would take for artifacts to show up on public repos, so here is a relatively generous retry strategy around the build. Once we have more data, we can tune these parameters.